Rakefile

```diff
@@ -22,6 +21,7 @@ PuppetLint.configuration.log_format = "%{path}:%{linenumber}:%{check}:%{KIND}:%{
 PuppetLint.configuration.send('relative')
 PuppetLint.configuration.send('disable_80chars')
 PuppetLint.configuration.send('disable_class_inherits_from_params_class')
+PuppetLint.configuration.send('disable_arrow_alignment')
```

Comment:
That's what I'm saying - the check will fail even with aligned arrows. It fails for me :) . But I can "properly" align the arrows and re-enable the check.